_includes/demos/proposal-csharp.cs

```diff
@@ -6,64 +6,55 @@
 using System.Net;
 using Newtonsoft.Json.Linq;
 /*
- * This is an example of using C# to view contract proposals via the Deriv/Binary API
- * This is a simple example where we do not check first if 
- * the user has the symbol available . To check this you would use the active_symbols call.
- * The example was originally written in dotnet version 5.0.  
- * it uses the Newtonsoft Json.net Library https://www.newtonsoft.com/json
- * This can be run on Windows/linux or Mac
+ * This is an example of using C# to view contract proposals via the Deriv/Binary API.
+ * This is a simple example where we do not check first if the user has the 
+ * symbol available. To check this you would use the active_symbols call.
+ * The example was originally written in dotnet version 5.0.
+ * It uses the Newtonsoft Json.net Library https://www.newtonsoft.com/json.
+ * This can be run on Windows/linux or Mac.
  *  
  * To run this example 
  * - Ensure you have dotnet SDK installed https://dotnet.microsoft.com/download/dotnet. 
- * - Run `dotnet new console --output proposal` In a dir on your computer
- * - Then run `dotnet add proposal package Newtonsoft.Json` 
- * - Change to the new proposal dir and edit the `Program.cs` file, paste in the contents of this script and save.
- * - Run `dotnet run --project proposal ` in the parent dir.  
- *
+ * - Run `dotnet new console --output proposal` in a directory on your computer.
+ * - Then run `dotnet add proposal package Newtonsoft.Json`. 
+ * - Change to the new proposal directory and edit the `Program.cs` file, paste in the contents of this script and save.
+ * - Run `dotnet run --project proposal` in the parent directory.
  */
 namespace DerivWSDemo
 {
     class DerivWS
     {
         private ClientWebSocket ws = new ClientWebSocket();
-        
-        private string app_id = "1089"; // Change this to yor app_id
+        private string app_id = "1089"; // Change this to yor app_id.
         private string websocket_url = "wss://ws.binaryws.com/websockets/v3?app_id=";
 
         public async Task SendRequest(string data)
         {
-
             while (this.ws.State == WebSocketState.Connecting) { };
             if (this.ws.State != WebSocketState.Open)
             {
                 throw new Exception("Connection is not open.");
             }
-
             var reqAsBytes = Encoding.UTF8.GetBytes(data);
-            var ticksRequest = new ArraySegment&lt;byte&gt;(reqAsBytes);
-
+            var ticksRequest = new ArraySegment&lt;byte>(reqAsBytes);
```

Comment:
Hmmm you changed the `>` sign but not the `&lt;`... 🤔

Reply:
Thanks for noticing, updated here.